tests/is/climate_HassClimateSetTemperature.yaml

```diff
@@ -1,6 +1,10 @@
-language: is
+language: "is"
 tests:
-  - sentences: []
+  - sentences:
+      - "Geturðu breytt hitastiginu í Svefnherberginu í 20°"
+      - "Stilltu Svefnherberginu hitastiginu í 20 gráður"
```

Comment:
What sentence structure are you trying to capture here?
Maybe add a comment on what this accomplishes or is testing in reality.